hazelcast/src/test/java/com/hazelcast/map/EntryProcessorBouncingNodesTest.java

```diff
@@ -28,108 +28,87 @@
 import com.hazelcast.query.Predicate;
 import com.hazelcast.query.PredicateBuilder;
 import com.hazelcast.test.AssertTask;
-import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastSerialParametersRunnerFactory;
 import com.hazelcast.test.HazelcastTestSupport;
-import com.hazelcast.test.TestHazelcastInstanceFactory;
 import com.hazelcast.test.annotation.ParallelTest;
 import com.hazelcast.test.annotation.QuickTest;
-import org.junit.After;
+import com.hazelcast.test.bounce.BounceMemberRule;
+import com.hazelcast.test.bounce.BounceTestConfiguration;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.junit.runners.Parameterized.UseParametersRunnerFactory;
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 
 /**
  * Creates a map that is used to test data consistency while nodes are joining and leaving the cluster.
- *
+ * <p>
  * The basic idea is pretty simple. We'll add a number to a list for each key in the IMap. This allows us to verify whether
  * the numbers are added in the correct order and also whether there's any data loss as nodes leave or join the cluster.
  */
-@RunWith(HazelcastSerialClassRunner.class)
+@RunWith(Parameterized.class)
+@UseParametersRunnerFactory(HazelcastSerialParametersRunnerFactory.class)
 @Category({QuickTest.class, ParallelTest.class})
 public class EntryProcessorBouncingNodesTest extends HazelcastTestSupport {
 
-    private static final int ENTRIES = 10;
-    private static final int ITERATIONS = 50;
+    private static final int ENTRIES = 50;
+    private static final int ITERATIONS = 100;
```

Comment:
With 50 iterations, test only had chance to restart the members once. Now there is enough time to restart nodes multiple times.